build.sbt

```diff
@@ -935,7 +991,7 @@ lazy val `project-manager` = (project in file("lib/scala/project-manager"))
     (Compile / run / connectInput) := true,
     commands += WithDebugCommand.withDebug,
     libraryDependencies ++= akka ++ Seq(akkaTestkit % Test),
-    libraryDependencies ++= circe,
+    libraryDependencies ++= circe ++ helidon,
```

Comment:
it needs **all** of those? :exploding_head:

Reply:
It is a way to put the classes on the module path. Although they are filtered out from the assembled JAR